src/utils/index.js

```diff
@@ -19,6 +20,128 @@ function logInfo(message) {
   console.log(blue(message));
 }
 
+function logHelp(command) {
+  const mainHelp = `
+Usage:
+  blazepack   Start dev server on existing project
+  blazepack   [options]  [command]
+
+Options:
+  --version   Output the current version
+  --port <p>  Specify port <p> for dev server
+
+Commands:
+  create          Create project from Template
+  start           Create project from Template & Start the dev server
+  clone           clone an existing codesandox
+  export          Export to codesandbox.io
+  install         Install dependency
+  add             Install dependency (alias for install)
+  uninstall       Remove dependency
+  remove          Remove dependency (alias for uninstall)
+  help [command]  show help for a particular command
+`;
+  if (!command) {
+    console.log(mainHelp);
+  } else {
+    switch (command) {
+      case 'create':
+        console.log(`
+Usage: blazepack create <name> [--template=<template>]
+        
+Create project with name <name> from Template <template>.
+If the --template option is not specified you will get the
+list of available templates to select from. 
+
+eg: blazepack create my-cra --template=react
+
+List of available templates are:`);
+        TEMPLATES.forEach(template => {
+          console.log(`  ${template}`);
+        });
+        break;
+      case 'start':
+        console.log(`
+Usage: blazepack start <name> [--template=<template>]
+        
+Create project with name <name> from Template <template> & also start the dev server.
+If the --template option is not specified you will get the list of available templates to select from. 
+
+eg: blazepack start my-angular-app --template=angular
+
+List of available templates are:`);
+        TEMPLATES.forEach(template => {
+          console.log(`  ${template}`);
+        });
+        break;
+      case 'clone':
+        console.log(`
+Usage: blazepack clone <url | embed-url | sandbox-id>
+
+Clone an existing codesandox.
+You can clone using a url or embed-url or sandbox-id.  
+
+eg: Clone from URL:
+blazepack clone https://codesandbox.io/s/use-undo-redo-yrts1
+
+eg: Cloning from Embed URL:
+blazepack clone https://codesandbox.io/embed/use-undo-redo-yrts1
+
+eg: Clone from Sandbox Id:
+blazepack clone use-undo-redo-yrts1
+`);
+        break;
+      case 'export':
+        console.log(`
+Usage: blazepack export [--open]
+
+Export your current project to codesandbox.io & also open the newly
+created sandbox in a browser tab if the --open option is specified.
+`);
+        break;
+      case 'install':
+        console.log(`\nUsage: blazepack install <package>`);
+      case 'add':
+        if (command === 'add') {
+          console.log(`\nUsage: blazepack add <package>`);
+        }
```

Comment:
How about we remove this condition, and log the description using a function like `logAddPackage()`